### PR TITLE
fix: Checkbox colors for active, hover and focussed states

### PR DESCRIPTION
**Pull request by SikindarMirza-p44**
Closes # 

## 📝 Description
Fixed the colours for Checkbox component following Figma for Active, Hover and Focussed states

> Add a brief description

## Screenshots

> Please provide screenshots for any visual changes

## Merge checklist

- [ ] Added/updated tests
- [ ] Added changeset

### CODE REVIEW

**Review by p44dyoachim — DISMISSED** [review]

File: packages/react/src/components/Checkbox/Checkbox.styles.ts
Comment (artlawry) [inline packages/react/src/components/Checkbox/Checkbox.styles.ts:63]: ah yeah, I suggested a variable for this and now I'm convinced - disabledOpacity or similar

Comment (partlysean44) [reply packages/react/src/components/Checkbox/Checkbox.styles.ts:63]: Agreed. This isn't the only place we'll be using a disabled state like that. Plus, it would make it easier to choose a value that's not seemingly so arbitrary (`0.38` for now is fine because that's what's in the Figma currently)

**Review by dmorton-p44 — APPROVED** [review]

**Comment by dmorton-p44** [comment]
Misread your slack - should wait for @p44dyoachim and @artlawry approval

**Review by p44dyoachim — APPROVED** [review]

**Review by artlawry — APPROVED** [review]

**Pull request merged**
